src/api/app/models/full_text_search.rb

```diff
@@ -13,7 +13,7 @@ class FullTextSearch
   self.ranker = :sph04
   self.per_page = 50
   self.star = false
-  self.max_matches = ThinkingSphinx::Configuration.instance.settings["max_matches"]
+  self.max_matches = 1000
```

Comment:
But this is not what is was before...

Reply:
> But this is not what is was before...

@hennevogel Sorry. Please have a look again.